deps/chakrashim/src/jsrtinspectorhelpers.cc

```diff
@@ -290,15 +335,59 @@ namespace jsrt {
                             &JsConvertValueToString, wasCopied);
   }
 
+  JsErrorCode InspectorHelpers::ArrayConcat(JsValueRef array, JsValueRef value) {
```

Comment:
One of the problems with jsrtutils is that it depends on libuv.  The standalone_inspector code currently does not and it didn't seem like a good idea to include an unnecessary dependency.  This was a way to work around that.